README.md

```diff
@@ -27,10 +27,10 @@ jupyter serverextension enable --py cs3api4lab --sys-prefix
 jupyter labextension install @sciencemesh/cs3api4lab
 ```
 
-To enable the Manager, the following configuration needs to be added to `jupyter_notebook_config.py`:
+To enable the Manager, the following configuration needs to be added to `jupyter_server_config.py`:
 
 ```python
-c.NotebookApp.contents_manager_class = 'cs3api4lab.CS3APIsManager'
+c.ServerApp.contents_manager_class = 'cs3api4lab.api.cs3apismanager.CS3APIsManager'
```

Comment:
Why did we remove `from cs3api4lab.api.cs3apismanager import CS3APIsManager` from `__init__`? Then we could still use the simplified version instead.

Reply:
The upgrade script messed up init.py a bit and I forgot about this trick. Fixed.